bundles/persistence/org.openhab.persistence.mysql/java/org/openhab/persistence/mysql/internal/MysqlPersistenceService.java

```diff
@@ -186,7 +186,7 @@ private String getTable(Item item) {
 			tableName = new String("Item" + rowId);
 			logger.debug("mySQL: new item {} is Item{}", itemName, rowId);
 		} catch (SQLException e) {
-			logger.error("mySQL: Could not create table for item '{}': ", itemName, e.getMessage());
+			logger.error("mySQL: Could not create item for '{}' in table 'Items' with statement '{}': {}", itemName, sqlCmd, e.getMessage());
```

Comment:
since 'item' is a "reserved" word i would change the statement to something like "could not create entry"